### DISCUSSION TITLE
Support for Stripe's new payment methods (Affirm, Cash App Pay, and Klarna)

### DISCUSSION THREAD

**Discussion by mpvosseller** [pay-rails/pay#1016] [Q&A] [answered]
Stripe notified us that:
> on August 1, 2024, we will automatically enable Affirm, Cash App Pay, and Klarna in your Checkout and Payment Links integrations.

Are you aware of any known issues the pay gem may have with these new payment methods in Stripe?

Thanks!

Screenshot of email:
<img width="877" alt="Screenshot 2024-07-01 at 5 25 14 PM" src="https://github.com/pay-rails/pay/assets/566863/d18930d8-155d-4786-9ab4-220d5529d585">

**Comment by excid3** [comment, marked as answer]
Nope, shouldn't affect our backend code. 👍

**Reply by mpvosseller** [reply]
Great to hear. Thank you.
